packages/framework-integration-tests/src/read-models/stock-read-model.ts

```diff
@@ -0,0 +1,17 @@
+import { ProjectionResult, UUID } from '@boostercloud/framework-types'
+import { Projects, ReadModel } from '@boostercloud/framework-core'
+import { Admin } from '../roles'
+import { Stock } from '../entities/stock'
+
+@ReadModel({
+  authorize: [Admin]
+})
+export class StockReadModel {
+  public constructor(readonly id: UUID, readonly warehouses: Record<string, number>) {
+  }
+
+  @Projects(Stock, 'id')
+  public static projectStock(entity: Stock): ProjectionResult<StockReadModel> {
+    return new StockReadModel(entity.id, entity.warehouses)
+  }
+}
```

Comment:
This read model is a one-to-one mapping with the Stock entity so that we can check the final status in the load test